lib/ui/addon/templates/components/toolbar-search-field.hbs

```diff
@@ -1,11 +1,19 @@
-{{input value=value type="text" placeholder="Search"}}
+<div class={{concat "toolbar__search " @wrapperClass}}>
+  <Input
+    @id={{this.inputId}}
+    @type="text"
+    @value={{@value}}
+    placeholder="Search"
+  />
+
+  {{#if @value}}
+    <button
+      class="toolbar__icon-button toolbar__search-clear-button js-search-field-clear-button"
+      title="clear"
+      {{on "click" this.clear}}
+    >
+      {{svg-jar "clear" width="14px" height="14px"}}
```

Comment:
Is `svg-jar` angle bracket compatible? If so, could we use `<SvgJar`?

Reply:
`svg-jar` is a helper.